app/init/init_gatekeeper.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from core import settings
+
+def register_apis_to_gatekeeper():
+
+    # Login
+    at = requests.post(
+        url=settings.GATEKEEPER_BASE_URL.unicode_string() + "api/login/",
+        headers={"Content-Type": "application/json"},
+        json={"username": "admin", "password": "admin"}
+    )
+
+    temp = at.json()
+
+    access = temp["access"]
+    refresh = temp["refresh"]
+
+    # Register APIs
+    # APIs shouldn't have slashes on their ends even though they do have them
+    apis_to_register = [
+        ("api/v1/data/upload", ["POST"], "pdm"), ("api/v1/pest-model",["POST", "GET"], "pdm"),
+        ("api/v1/rule", ["GET", "POST", "DELETE"], "pdm"),
+        ("api/v1/tool/calculate-risk-index/weather/{weather_dataset_id}/model/{model_ids}/verbose", ["GET"], "pdm"),
+        ("api/v1/tool/calculate-risk-index/weather/{weather_dataset_id}/model/{model_ids}/high", ["GET"], "pdm")
+    ]
+
+    for atr in apis_to_register:
+        requests.post(
+            url=settings.GATEKEEPER_BASE_URL.unicode_string() + "api/register_service/",
```

Comment:
why are you using the method `unicode_string()` does it provide any more sanitazation of urls?

Reply:
I think variable as default parsed as strings. Additionally, you can always use `str()` method